SportsHub/SportsHub.DAL/SportsHub.DAL.csproj

```diff
@@ -17,6 +17,7 @@
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
```

Comment:
Why do you need this package? Is it possible to use System.Text.Json?

Reply:
Updated to use System.Text.Json.